### DISCUSSION TITLE
Using the officially recommended cross env configuration environment variables, but did not receive the correct ENV during packaging 使用官方推荐的cross-env配置环境变量，但是在打包的时候并未拿到正确的ENV

### DISCUSSION THREAD

**Discussion by GUJUNNIAN** [umijs/dumi#2118] [Q&A]
I used the officially recommended cross env configuration environment variable, but did not receive the correct ENV during packaging. What I hope is to dynamically change the packaged input and output based on different configuration variables
![image](https://github.com/umijs/dumi/assets/30283164/56704051-4b9f-4d94-b79a-19c32694d9f4)
![image](https://github.com/umijs/dumi/assets/30283164/82fc4596-8b58-408f-9465-76500ca6ccb0)

**Comment by PeachScript** [comment]
换个名字，`NODE_ENV` 框架会内部设置
